Multiple destination instance update (#4856)
segmentio/segment-docs: src/connections/destinations/add-destination.md

You can create unique destination filters for each destination instance connected to the same source.

> >info ""

Reviewer: ```suggestion
> info ""
```

> >Some destinations do not support having multiple instances connected to the same source. In that case, the option to add a second instance of that destination does not appear.

Reviewer: ```suggestion
> Some destinations don't support multiple instances connected to the same source. If this is the case, you won't see the option to add a second instance of that destination.
```
